src/TrackRowInfoVisQuantitativeBar.js

```diff
@@ -92,10 +112,10 @@ export default function TrackRowInfoVisQuantitativeBar(props) {
                 const barTop = yScale(i);
                 let currentBarLeft = (isLeft ? width : 0);
 
-                field.forEach((f, i) => {
-                    const barWidth = xScale(d[f]);    
-                    const color = colorScale(f);
-
+                field.forEach((f, j) => {
```

Comment:
```suggestion
                field.forEach((f) => {
```

It looks like the j variable is never used